homeassistant/components/spotify/media_player.py

```diff
@@ -299,6 +301,13 @@ def play_media(self, media_type: str, media_id: str, **kwargs) -> None:
             kwargs["uris"] = [media_id]
         elif media_type == MEDIA_TYPE_PLAYLIST:
             kwargs["context_uri"] = media_id
+            if self._start_random:
+                response = self._spotify.playlist_tracks(media_id,
```

Comment:
What if it isn't a playlist, but al album?
```suggestion
                response = self._spotify.playlist_tracks(media_id,
```

Reply:
I do not think the current integration supports albums, there is only MEDIA_TYPE_MUSIC and MEDIA_TYPE_PLAYLIST, I pressumed the MEDIA_TYPE_MUSIC is a single song?

Do you know how that works

Reply:
There isn't a difference in the service call.
Maybe it is even better at this point to ignore the `media_type` completely and decide based on the Spotify URI what it is.

This same functionality can be used for artists, playlists and albums.

Reply:
I will test albums and artist tonight and look if it can handle it.

Reply:
@frenck albums and artist did not work, so I commited a fix to check the uri for "playlist"